recipes/cmudict/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cmudict" %}
+{% set version = "1.0.32" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cmudict-{{ version }}.tar.gz
+  sha256: e84a587bb610b3a837a93f07494e874860cf205ea7f23db652b871093a699f38
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python >={{ python_min }},<4.0
+    - importlib-metadata >=5
+    - importlib-resources >=5
+
+test:
+  imports:
+    - cmudict
+  commands:
+    - pip check
+  requires:
+    - pip
+    - python {{ python_min }}
+
+about:
+  home: https://github.com/prosegrinder/python-cmudict
+  summary: A versioned python wrapper package for The CMU Pronouncing Dictionary data files.
+  license: GPL-3.0 AND BSD-2-Clause
```

Comment:
To satisfy the linter, I think you may need to update the `GPL` one (I'm not sure if it is that or the `AND`):
https://spdx.org/licenses/

Reply:
@synapticarbors - Fixed !